storybook/stories/grid/resolution.md

```diff
@@ -0,0 +1,21 @@
+`show` allows you to define when a given component is visible based on the screen width.
```

Comment:
I think having a hide prop in addition to show would be helpful.

Reply:
That's a good idea. I think as far as we don't allow having `show` and `hide` set at the same time there should be no issues. I'll take a look if I don't add it here I may create a ticket for a follow up PR

Reply:
I've added https://github.com/obartra/reflex/issues/307